Doc: Prepare for 1.11.0 release (#8914)
OpenTTD/OpenTTD: changelog.txt

1.11.0 (2021-04-01)
------------------------------------------------------------------------
Feature: Allow setting a custom terrain type to define highest peak (#8891)
Feature: Auto-detect map height limit based on generated map (#8891)
Feature: Setting to indicate desert coverage for tropic climate and snow coverage for arctic climate (replaces snow line height) (#8891)
Add: Allow setting the highest mountain for heightmaps (#8891)
Change: Scale exported heightmaps to highest peak and inform the user of this value (#8891)
Change: Remove "maximum map height" from the New Game GUI (#8891)

> Fix: OpenGL performance with some AMD GPUs (#8876)

Reviewer: wrong order :p

1.11.0 (2021-04-01)
------------------------------------------------------------------------
Feature: Allow setting a custom terrain type to define highest peak (#8891)
Feature: Auto-detect map height limit based on generated map (#8891)
Feature: Setting to indicate desert coverage for tropic climate (#8891)

> Feature: Setting to indicate snow coverage for arctic climate (replaces snow line height) (#8891)

Reviewer: merge
```suggestion
Feature: Setting to indicate desert coverage for tropic climate and snow coverage for arctic climate (replaces snow line height) (#8891)
```

Add: Allow setting the highest mountain for heightmaps (#8891)

> Change: Scale heightmaps we export to highest peak and inform the user of this value (#8891)

Reviewer: ```suggestion
Change: Scale exported heightmaps to highest peak and inform the user of this value (#8891)
```

Change: Remove "maximum map height" from the New Game GUI (#8891)

> Change: Do not disallow persistent buffer mapping on AMD GPUs, as it is actually faster (#8876)

Reviewer: Might be worth changing this to a "fix", since that's how most people will perceive it ("Fix: OpenGL performance with some AMD GPUs")

1.11.0 (2021-04-01)
------------------------------------------------------------------------
Feature: Allow setting a custom terrain type to define highest peak (#8891)
Feature: Auto-detect map height limit based on generated map (#8891)
Feature: Setting to indicate desert coverage for tropic climate (#8891)
Feature: Setting to indicate snow coverage for arctic climate (replaces snow line height) (#8891)
Add: Allow setting the highest mountain for heightmaps (#8891)
Change: Scale heightmaps we export to highest peak and inform the user of this value (#8891)
Change: Remove "maximum map height" from the New Game GUI (#8891)
Change: Do not disallow persistent buffer mapping on AMD GPUs, as it is actually faster (#8876)
Fix #8565: Stopped road vehicle displays a speed different than 0 (#8901)
Fix #8886: Don't try to resolve folders within tars named '.' (#8893)

> Fix: placing random trees in SE crashes the game (#8892)

Reviewer: ```suggestion
Fix: Placing random trees in SE crashes the game (#8892)
```

Fix #8875: Filter string in station window breaks flow in user interface (#8885)

> Fix: [OpenGL] Broken window resizing due to invalid buffer pitch on texture creation (#8883)

Reviewer 2: That's a fix for the fix just below and thus doesn't need to be there.
